start-site/src/main/resources/application.yml

```diff
@@ -142,12 +142,14 @@ initializr:
         versionProperty: spring-cloud-services.version
         additionalBoms: [spring-cloud]
         mappings:
-          - compatibilityRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"
+          - compatibilityRange: "2.0.x.RELEASE"
```

Comment:
Can you share why you've changed all compatibility ranges like this? Please review [the documentation](https://docs.spring.io/initializr/docs/current/reference/html/#dependencies-compatibility-range).

This is an open range to start with the latest `2.0.x.RELEASE` and any version after that (including `2.1.x`, `2.2.x`, etc). That's definitely not what we want here. Please revert and add the range for 2.3.x consistently with what we have now.

Reply:
Thanks for the link. I had no idea these docs existed!